src/instantsearch/widget.test.js

```diff
@@ -34,23 +34,28 @@ describe('instantsearch widget', () => {
   it('configures the helper', () => {
     const widget = algoliaPlacesWidget(defaultOptions);
     widget.init({ helper });
+    expect(helper.setQueryParameter).toBeCalledWith('insideBoundingBox');
```

Comment:
I think we can improve the test and use an expected state instead of relying on methods being called.

Reply:
Agree, I will change that tomorrow.